operator/cmd/mesh/manifest-apply.go

```diff
@@ -19,12 +19,21 @@ import (
 	"os"
 	"time"
 
+	"github.com/spf13/cobra"
+
+	"istio.io/api/operator/v1alpha1"
+	iopv1alpha1 "istio.io/istio/operator/pkg/apis/istio/v1alpha1"
 	"istio.io/istio/operator/pkg/kubectlcmd"
 	"istio.io/istio/operator/pkg/manifest"
 	"istio.io/istio/operator/pkg/name"
+	"istio.io/istio/operator/pkg/translate"
 	"istio.io/istio/operator/version"
+)
 
-	"github.com/spf13/cobra"
+const (
+	// installedSpecCRName is the name of the IstioOperator CR stored in the cluster which is a copy of the CR used
+	// in the last manifest apply operation. It is used to store the current installed cluster state.
+	installedSpecCRName = "installed"
```

Comment:
This is incompatible with multiple control planes

Reply:
Because there are multiple CRs that are "installed"

Reply:
I've put in a placeholder to populate with revision when it's integrated.